extensions/schema-compare/src/test/schemaCompare.test.ts

```diff
@@ -634,8 +634,8 @@ describe('SchemaCompareMainWindow: Button clicks', function (): void {
 		//Verify that switch actually happened
 		should.notEqual(result.sourceEndpointInfo, undefined);
 		should.equal(result.sourceEndpointInfo.endpointType, mssql.SchemaCompareEndpointType.Database);
-		should.equal(result.sourceEndpointInfo.serverName, 'My Server');
-		should.equal(result.sourceEndpointInfo.databaseName, 'My Database');
+		should.equal(result.sourceEndpointInfo.serverName, '');
```

Comment:
why did these get changed to empty strings?

Reply:
changed back